### ISSUE TITLE
P adjustment for multiple comparisons

### ISSUE THREAD

**Issue by Vh2m** [0todd0000/spm1d#117]
Hi Todd.
I am performing some SPM paired t.test to test if there are differences in the lower limb's joint kinematics (A-K-H) flexion for stepping under 2 different ground conditions. I got more than one supra threshold cluster and some of them are quiet close to each other. So trying to be not that much conservative I am wondering if it is possible to perform a Holm-Bonferroni correction for the resulting p-values instead of Bonferroni adjusting the alpha a-priori. If so, would you recommend to do it? could you guide me how to do it? 
Thanks in advance!

**Comment by 0todd0000** [comment]
Hello!  There is no need to correct for multiple comparisons in cluster-level inference. Corrections only need to be applied (a) to field-level inference, and (b) when performing multiple tests, like multiple t tests.

* Field-level inference: this is the hypothesis test. Random field theory (RFT) is used to control alpha=0.05, and the result is a critical threshold (e.g. a critical t value). Smooth, Gaussian random fields would produce a t field that traverses this threshold in only alpha % of an infinite number of experiments (when the null hypothesis is true). A correction for multiple comparisons is needed only when performing multiple tests. A multiple comparisons correction will cause the critical threshold to rise.

* Cluster-level inference: this can be regarded as a type of *post hoc* analysis, which qualifies the main test's critical threshold results. These p values represent the probability that smooth, Gaussian random fields would produce suprathreshold clusters of the given size (when the null hypothesis is true). If a multiple comparisons correction is applied to field-level inference, then the threshold rises, so this correction propagates to cluster-level inference.

In other words, multiple clusters do not imply multiple tests.

Todd

**Comment by Vh2m** [comment]
Hi todd! 
Thanks for the explanation.!
